docs/references/commands.txt

```diff
@@ -0,0 +1,326 @@
+
+pro collect-logs
+================
+
+**Usage:**
+
+``pro collect-logs [-h] [-o OUTPUT]``
+
+**Description:**
+
+Collect logs and relevant system information into a tarball.
+
+pro config
+==========
+
+**Usage:**
+
+``pro config [-h] {show,set,unset} ...``
+
+**Description:**
+
+Manage Ubuntu Pro configuration
+
+pro config show
+---------------
+
+**Usage:**
+
+``pro config show [-h] [key]``
+
+**Description:**
+
+Show customizable configuration settings
+
+pro config set
+--------------
+
+**Usage:**
+
+``pro config set [-h] key_value_pair``
+
+**Description:**
+
+Set and apply Ubuntu Pro configuration settings
+
+pro config unset
+----------------
+
+**Usage:**
+
+``pro config unset [-h] key``
+
+**Description:**
+
+Unset Ubuntu Pro configuration setting
+
+pro detach
+==========
+
+**Usage:**
+
+``pro detach [-h] [--assume-yes] [--format {cli,json}]``
+
+**Description:**
+
+Detach this machine from an Ubuntu Pro subscription.
+
+pro disable
+===========
+
+**Usage:**
+
+``pro disable [-h] [--assume-yes] [--format {cli,json}] [--purge] service [service ...]``
+
+**Description:**
+
+Disable an Ubuntu Pro service.
+
+pro enable
+==========
+
+**Usage:**
+
+``pro enable [-h] [--assume-yes] [--access-only] [--beta] [--format {cli,json}] [--variant VARIANT] service [service ...]``
+
+**Description:**
+
+Activate and configure this machine's access to an Ubuntu Pro service.
+
+pro fix
+=======
+
+**Usage:**
+
+``pro fix [-h] [--dry-run] [--no-related] security_issue``
+
+**Description:**
+
+Inspect and resolve CVEs and USNs (Ubuntu Security Notices) on this machine.
+
+The exit code will be:
+    0: the fix was successfully applied or the system is not affected
+    1: the fix cannot be applied
+    2: the fix was applied but requires a reboot before it takes effect
+
+pro help
+========
+
+**Usage:**
+
+``pro help [-h] [--format {tabular,json,yaml}] [--all] [service]``
+
+**Description:**
+
+Provide detailed information about Ubuntu Pro services.
+
+pro refresh
+===========
+
+**Usage:**
+
+``pro refresh [-h] [{contract,config,messages}]``
+
+**Description:**
+
+Refresh three distinct Ubuntu Pro related artifacts in the system:
+
+    * contract: Update contract details from the server.
+    * config:   Reload the config file.
+    * messages: Update APT and MOTD messages related to UA.
+
+You can individually target any of the three specific actions,
+by passing the target name to the command.  If no `target`
+is specified, all targets are refreshed.
+
+
+pro security-status
+===================
+
+**Usage:**
+
+``pro security-status [-h] [--format {json,yaml,text}] [--thirdparty | --unavailable | --esm-infra | --esm-apps]``
+
+**Description:**
+
+Show security updates for packages in the system, including all
+available Expanded Security Maintenance (ESM) related content.
+
+Shows counts of how many packages are supported for security updates
+in the system.
+
+If called with --format json|yaml it shows a summary of the
```

Comment:
```suggestion
If called with ``--format json|yaml`` it shows a summary of the
```

Reply:
This has the same issue as above, with no backticks the `--` is rendered as a single em-dash. Would this work?:

'If the format called is JSON or YAML...'